pkgs/applications/system/booster/default.nix

```diff
@@ -0,0 +1,55 @@
+{ bash
+, binutils
+, buildGoModule
+, fetchFromGitHub
+, kbd
+, lib
+, libfido2
+, lvm2
+, lz4
+, makeWrapper
+, mdadm
+, unixtools
+, xz
+, zfs
+}:
+
+buildGoModule rec {
+  pname = "booster";
+  version = "0.9";
+
+  src = fetchFromGitHub {
+    owner = "anatol";
+    repo = pname;
+    rev = version;
+    hash = "sha256-kalVFVBb+ngoUpm+iiIHGS6vBVLEvTVyKuSMSMbp7Qc=";
+  };
+
+  vendorHash = "sha256-GD+nsT4/Y2mTF+ztOC3N560BY5+QSfsPrXZ+dJYtzAw=";
+
+  postPatch = ''
+    substituteInPlace init/main.go --replace "/usr/bin/fsck" "${unixtools.fsck}/bin/fsck"
+  '';
+
+  # integration tests are run against the current kernel
+  doCheck = false;
+
+  buildInputs = [ kbd lz4 xz ];
+
+  nativeBuildInputs = [ makeWrapper ] ++ buildInputs;
```

Comment:
Why do we need them in nativeBuildInputs?

Reply:
because we need the binaries for integration testing; would it be better to just add them to `nativeBuildInputs`?